### ISSUE TITLE
Can you start/stop?

### ISSUE THREAD

**Issue by bhalperin** [jonahvsweb/touchpoint-js#3]
Cool stuff!
Can you start/stop it programmatically? Can be very useful i.e. if I have a web app and wish to video some action of the real app, maybe by discretely tap somewhere to start, then record, then stop so app can continue w/o the effect. I can leave the script reference in the app in production too then.
Thanks

**Comment by jonahvsweb** [comment]
@bhalperin Yup, I've also been thinking about adding that functionality. Planning on mapping it to a hotkey(s) of some sort to give the option to turn on and off on the fly. Hoping to get to this soon. Will update this ticket when I push the update. Thanks and hope TouchPoint has been useful to you.
